docs/setup/setup-bicep.md

```diff
@@ -135,3 +127,27 @@ configuration:
   [4]: configuring-expansion.md#bicepsourceexpansion
   [5]: ../using-bicep.md#usingparameterfiles
   [6]: ../using-templates.md#bymetadata
+
+### Configuring timeout
+
+[:octicons-book-24: Docs][7]
+
+In certain environments it may be necessary to increase the default timeout for building Bicep files.
+This can occur if your Bicep deployments are:
+
+- Large and complex.
+- Use modules restored from a registry.
```

Comment:
Maybe we can also mention nested modules here? Probably goes in the _Large and Complex_ category, but worth being explictit here I think.